src/tool_application_support.cpp

```diff
@@ -24,23 +25,61 @@
 
 namespace clangmetatool {
 
+namespace {
+
+#ifndef CLANG_INSTALL_LOCATION
+#define CLANG_INSTALL_LOCATION ""
```

Comment:
Should I issue a pre-processor warning for people who choose to build this but forget to specify this as a  '-D' option?

Reply:
There's also the `libc++` containing directory at `$(llvm-config --prefx)/include/v1/c++`, which the toolchain knows how to approach if we call [`AddClangCXXSystemIncludeArgs`](https://clang.llvm.org/doxygen/classclang_1_1driver_1_1ToolChain.html#adb8a01dbb138e8c9494d9e552b6aacad). This seems fine for our use case since this should be API compatible for `libstdc++`.

Reply:
I think that we can assume they are using the provided build scripts, which set this preprocessor predefine. You can probably even remove the ifndef checks